ui/apps/platform/cypress/helpers/request.js

```diff
@@ -1,35 +1,59 @@
+/*
+ * For pages which have GraphQL and REST requests.
+ *
+ * For example, given 'searchOptions' return:
+ * {
+ *     method: 'POST',
+ *     url: '/api/graphql?opname=searchOptions',
+ * }
+ */
+export function getRouteMatcherForGraphQL(opname) {
+    return {
+        method: 'POST',
+        url: `/api/graphql?opname=${opname}`,
+    };
+}
+
+/*
+ * For pages which have only GraphQL requests.
+ *
+ * For example, given ['searchOptions', 'getDeployments'] return:
+ * {
+ *     searchOptions: {
+ *         method: 'POST',
+ *         url: '/api/graphql?opname=searchOptions',
+ *     },
+ *     getDeployments: {
+ *         method: 'POST',
+ *         url: '/api/graphql?opname=getDeployments',
+ *     },
+ * }
+ */
+export function getRouteMatcherMapForGraphQL(opnames) {
+    const routeMatcherMap = {};
+
+    opnames.forEach((opname) => {
+        routeMatcherMap[opname] = getRouteMatcherForGraphQL(opname);
+    });
+
+    return routeMatcherMap;
+}
+
 /*
```

Comment:
Aha, so that is the meaning of the second star! Thank you for critique and pictures.

Ouch, I missed that nuance when we rewrote service functions in TypeScript.

Reply:
With `/**` Visual Studio Code interprets brackets `[routeMatcherMap]` as optional.

I had searched that specific detail online but missed the obvious notation.